pages/careers/_open-positions.html

```diff
@@ -3,27 +3,29 @@
     <h2>Open Positions</h2>
     <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're currently looking for:</p>
     <h3>Senior Full-Stack Engineer</h3>
+    <h4>What You'll Work On</h4>
+    <ul>
+      <li>Engineering a fundamentally better DevOps experience.</li>
+    </ul>
     <h4>What You'll Do</h4>
     <ul>
       <li><strong>Work on just about every aspect of the Gruntwork stack.</strong> Write Go-based CLI tools, work on Serverless apps in Node, write Terraform modules, write Python scripts and Bash scripts, do frontend development, learn the newest tech from AWS, invent new ways of managing infrastructure. We're looking for true full-stack engineers who can go deep in any area. We're less concerned with prior experience than we are with demonstrated ability to go deep when necessary.</li>
     </ul>
     <h4>Your Ideal Background</h4>
     <ul>
-      <li>Demonstrated experience building good software.</li>
-      <li>Bonus points for visible open source work.</li>
-      <li>Bonus points for any technical presentations you've done in the past.</li>
-      <li>Bonus points for a blog.</li>
-      <li>Bonus points for something in your profile that makes us smile.</li>
-      <li>To be clear, you don't have to be good at everything. Our most important criteria are:
+      <li>You know how to write code across the stack ("Dev").</li>
+      <li>You have experience running production software ("Ops").</li>
+      <li>You have a strong background in software engineering.</li>
+      <li>Bonus points for:
```

Comment:
My worry is that this list, as presented, will scare off candidates who don't have these. I mean, the other things that could be part of this list are strong previous employers, top-tier schools, and personal references, but again, none of those are required. So it's not obvious to me how much we gain by listing these, versus how much we may lose.

Perhaps instead of listing each item as a separate bullet point

Reply:
I guess I don't feel that strongly about it, so if it makes you worried, I'm inclined to remove. Thoughts?

Reply:
Haha, just realized I didn't quite finish typing that previous comment, did I? Sorry, multi-tasking a bit too much today.

I like the idea of providing more info on what we're looking for. In a sense, I want to say that these are the things we consider, but none are required (including, crucially, things like "top-tier school" and "CS degree" that are stupidly required by so many other jobs). However, I'm not sure how to do it in a way that is genuine and candidates won't misinterpret as an intimidating checklist of "must-haves."

So for now, removing is fine, but I'm definitely open to other ways of phrasing this.

Reply:
Ok, simplified this in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/37/commits/f22c3c7cdcc02a2bf37853ef417e1ebfac445a9f and pushed it live. Merging now!